assignments/arrays.js

```diff
@@ -63,34 +63,64 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+
+
+console.log(obinventor[32]);
+
```

Comment:
compare this to the example console.log you started out with:
```console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );```
as of right now you're logging the entire object instead of it being dynamic. take a look again at the interpolated string syntax that uses backticks ( `string` instead of 'string' or "string") and how to interpret dynamic values in that string with the `${JS values}` syntax.
ex: ``` `Car 33's year is ${inventory[32].car_year}` ```